examples/azure_file/azure.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: azure
+spec:
+ containers:
+  - image: kubernetes/pause
+    name: azure
+    volumeMounts:
+      - name: azure
+        mountPath: /mnt/azure
+ volumes:
+      - name: azure
+        azureFile:
+          accountName: account_name
+          accountKey: account_key
```

Comment:
I agree with @swagiaal.  We should not embed the account info directly into the volume.

Why not a CloudProvider like aws/gce/cinder?